test/using_witnet.js

```diff
@@ -24,7 +24,8 @@ contract("UsingWitnet", accounts => {
     before(async () => {
       witnet = await Witnet.deployed()
       wrb = await WRB.new([accounts[0]])
-      wrbProxy = await WRBProxy.new(wrb.address)
+      wrbProxy = await WRBProxy.deployed()
```

Comment:
```suggestion
      wrbProxy = await WRBProxy.new()
```

Otherwise it only works the first time you run it, but not the second. :sweat_smile:

Reply:
Excellent, nice !